tests/db_migrator_input/minigraph.xml

```diff
@@ -19,9 +19,17 @@
     </Devices>
    </PngDec>
   <MetadataDeclaration>
-    <Devices xmlns:a="http://schemas">
+    <Devices xmlns:a="http://schemas.datacontract.org/2004/07/Microsoft.Search.Autopilot.Evolution">
       <a:DeviceMetadata>
-        <a:Properties/>
+        <a:Name>SONiC-Dummy</a:Name>
+        <a:Properties>
+          <a:DeviceProperty>
+            <a:Name>DnsNameserverResources</a:Name>
```

Comment:
>DnsNameserverResources [](http://example.com/codeflow?start=20&length=22)

Block this PR until minigraph schema is accepted by our internal team.

Reply:
I think minigraph schema has been accepted.

Reply:
Actually, we should revert minigraph schema code change and test cases.